app/views/admin/administrators/administrators/delete.html.erb

```diff
@@ -1,6 +1,8 @@
-<h1>Do you want to delete this user?</h1>
-<p class="govuk-!-font-weight-bold">Admin user: <span class="govuk-!-font-weight-regular"><%= @administrator.full_name %></span></p>
+<h1>Confirm deletion?</h1>
+<p class="govuk-body-l">Once deleted, <%= @administrator.full_name %> won't be recoverable.</p>
```

Comment:
I didn't really know what else to write here, but `Admin user: [user]` felt clunky to me

Reply:
`Once deleted, <%= @administrator.full_name %>'s account won't be recoverable`?

Reply:
Hah, it's the `won't be recoverable` I thought was a bit weird (isn't it technically not even true?), but yeah, that's a good suggestion

Reply:
Well, there is no way for the user to recover it yet, soooo, it's not technically lying :D

Reply:
This was exact text that Shani had requested - double-check with her if the text is correct